.github/workflows/CI_CD.yml

```diff
@@ -0,0 +1,84 @@
+name: CI/CD
+
+on:
+  push:
+    branches: [ main, dev ]
+  pull_request:
```

Comment:
We should probably not deploy PRs to Heroku by default, as we have limited Heroku Dyno hours available (Even deploying *main* and *dev* on the same account might lead to issues later, we should keep an eye on that)

Reply:
But we want to apply CI to PRs right? If so, I will add a safeguard below, in the deployment job that catches PRs and only deploys on pushes. 

What do you mean by saying "We should keep an eye on that"? Just deploy both main and dev and see what happens or deploy only main for safety?

Reply:
> But we want to apply CI to PRs right?

Yes, please :)

> If so, I will add a safeguard below, in the deployment job that catches PRs and only deploys on pushes.

That'd be great

> What do you mean by saying "We should keep an eye on that"?

Check on Heroku to see what percentage of free hours are used up, e.g. in the middle of the month.

> Just deploy both main and dev and see what happens or deploy only main for safety?

Let's deploy both. We'll see what happens